src/test/java/features/327 Download Verification Steps.feature

```diff
@@ -25,4 +25,11 @@ Feature: Download Verification Steps
     When I navigate to the Upload Page
       And I re-upload the most recently downloaded file to the choose file button
     Then I verify the file counter with the attribute data-filecount has the value 1
-      And I verify the upload result 1 with the attribute data-filename has the value TestPDF.pdf
\ No newline at end of file
+      And I verify the upload result 1 with the attribute data-filename has the value TestPDF.pdf
```

Comment:
This seems confusingly worded. Doesn't need to change now, but something to discuss for the future.